### PR TITLE
DOMA-3976 fix ticket last comment filter logic

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
We need to filter tickets by the date of the last comment created by a resident, and not any last comment.
Rewrote the logic, updated the old tickets in the migration.

### CODE REVIEW

File: apps/condo/domains/ticket/schema/Ticket.js [hunk 1]
```diff
@@ -93,8 +93,8 @@ const Ticket = new GQLListSchema('Ticket', {
             schemaDoc: 'When status of the ticket was changed to completed',
             type: DateTimeUtc,
         },
-        lastCommentAt: {
-            schemaDoc: 'Last comment time in ticket',
+        lastResidentCommentAt: {
```
Comment (pahaz) [inline apps/condo/domains/ticket/schema/Ticket.js:96]: Why we need this field? we already have TicketCommentsTime

Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/Ticket.js:96]: this field is needed to filter tickets by the date of the last comment of a resident

**Review by AntonAL — COMMENTED** [review]
If you do need to ignore new field in `TicketChange`, add it to `OMIT_TICKET_CHANGE_TRACKABLE_FIELDS` array.

File: apps/condo/domains/ticket/schema/Ticket.js (hunk 1 above)
Comment (pahaz) [reply apps/condo/domains/ticket/schema/Ticket.js:96]: It’s better to create two separate field for that

Comment (nomerdvadcatpyat) [reply apps/condo/domains/ticket/schema/Ticket.js:100]: done

**Review by dkoviazin — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

**Pull request merged**